### ISSUE TITLE
Voting for new logo

### ISSUE THREAD

**Issue by lunny** [go-gitea/gitea#1516]
@qdbdbp @kolaente Let's vote for Gitea's new Logo. First reply is for @kolaente and the second is for @qdbdbp and other replies before that will be deleted. The vote duration is from now until v1.2 release (about 10 days) I think or we have to put this to v1.3.

**Comment by strk** [comment]
Are @qdbdbp and @kolaente supposed to add their proposals here as comments ?

**Comment by bkcsoft** [comment]
Logo by @kolaente https://github.com/go-gitea/gitea/issues/1312#issue-215120549 ![](https://cloud.githubusercontent.com/assets/13721712/24114544/f0d3cb4c-0da0-11e7-8ac6-f60b36808018.png)

**Comment by bkcsoft** [comment]
Logo by @qdbdbp https://github.com/go-gitea/gitea/issues/1260#issuecomment-290051119
![](https://cloud.githubusercontent.com/assets/19255071/24450561/a14d7564-147b-11e7-8ae4-10ec46479cce.jpg)

**Comment by bkcsoft** [comment]
Just a quick heads-up on both though... Using the Git-logo in our logo is actually **not legal**: https://git-scm.com/trademark ... And technically neither is using the term "Git" in the name 🙄 

So before starting to use this logo, we should send an email to trademark@sfconservancy.org asking if it's okey...

**Comment by agaida** [comment]
But the SF are more or less nice people, they allowed the usage for gitolite, GitLab, but one should ask for the written permission

**Comment by lunny** [comment]
@strk please comment after that two vote replies. So I deleted your comment.

**Comment by bkcsoft** [comment]
@agaida Both of those have been [Grandfathered](https://en.wikipedia.org/wiki/Grandfather_clause), and none of them use the git-logo in their logo (and never have)

Also from Gitolites website: http://gitolite.com/gitolite/index.html#license
> Note:
> GIT is a trademark of Software Freedom Conservancy and **my use of "Gitolite" is under license.**

**Comment by andreynering** [comment]
So, I think we will have to rewrite the tea tag to remove the Git reference. Is there any other generic symbol that refer to VCS?

**Comment by jhasse** [comment]
I would wait for the response from SF first.

**Comment by kifirkin** [comment]
I think VS Code changed git icon to this one because of license issues

![git](https://cloud.githubusercontent.com/assets/6573628/25228803/728aef88-25d6-11e7-83d1-c003f574d6a4.PNG)

**Comment by jhasse** [comment]
> because of license issues

Do you have a source for this? I think it was because they added support for other VCSs, e.g. SVN.

**Comment by kifirkin** [comment]
@jhasse No I have no source of this icon :disappointed:

**Comment by kolaente** [comment]
@alexmironof i think he means a proof, not a source file

**Comment by gravemalte** [comment]
@jhasse They just improved it. No license issue.

Source: https://github.com/Microsoft/vscode/issues/18675#issuecomment-289680726

**Comment by pgaskin** [comment]
@bkcsoft 

https://git-scm.com/trademark

> In addition, you may not use any of the Marks as a syllable in a new word or as part of a portmanteau (e.g., "Gitalicious", "Gitpedia") used as a mark for a third-party product or service without Conservancy's written permission. For the avoidance of doubt, this provision applies even to third-party marks that use the Marks as a syllable or as part of a portmanteau to refer to a product or service's use of Git code.

**Comment by kolaente** [comment]
So, we need a new name?

**Comment by jhasse** [comment]
> So, we need a new name?

Or Conservancy's written permission.

**Comment by bkcsoft** [comment]
@jhasse @geek1011 @kolaente Just to clarify, I'm _not_ saying we should change the name. I'm just saying that we should _not_ put the git-logo in our logo, since that would just put us _more_ in violation with their trademark.

And the "punchline" `Git with a cup of tea` can still live on, even if the name might change in the future 😉

**Comment by priyadarshan** [comment]
Like for many things, the name is an important aspect of an application. A name definitely can influence its development and future, in some ways, practical or more subtle.

"Gitea" is a nice name, still it seems to me it does not reflect the broader and deeper aspect of what it offers.

In my humble opinion, "Gitea" is cute and entertains of course an echo of culinary culture, perhaps also hinting to its possible social nature, but it does not fully convey its potential.

I am afraid an application of a much broader nature and so promising like this is would be negatively affected by a wrong name.

Unfortunately, to some cultures, even the name [gogs](https://en.wikipedia.org/wiki/Gog_and_Magog) might have been not a good choice.

This project would deserver a name deeply connected with its nature, with a short and nice sound (although two syllables do help form a rhythm), and it should have a unique feeling, charged with newness and enthusiasm , like its vision and its community.

Please consider changing the name of gitea. This is the right time, the beginning of a new phase, when the application is allowed to unfold more of its potential. Or, at least allow for more choices to come in, so that a freer deliberation could be made by the maintainers.

In any case, thank you so much for this wonderful app. It is truly a first-class app.

**Comment by jcgruenhage** [comment]
Another way would be to just take a branch of tea leafs, but I have no idea how to make that into a nice flat icon ;D

**Comment by jcgruenhage** [comment]
@priyadarshan even now, changing the name will require a lot of work, since that would mean a lot of code changes, a new domain, moving this repo, and so on.

**Comment by priyadarshan** [comment]
A name is such a vital aspect to an application, that perhaps should be seen as a long term investment, worth the extra work. Of course, the more the wait, the more work it would require. Still, it should not be taken lightly, or hurriedly.

As per the domain name change, I pledge to donate $100 to support an eventual domain change, just let me know, in due time, where to donate.

I am just a very new, grateful user of gitea. I will be happy with whatever decision the maintainers will support.

**Comment by jcgruenhage** [comment]
I forgot the hardest part: Coming up with a good name. Anyone got any suggestions?

**Comment by priyadarshan** [comment]
Thank you for the chance. A possibility:
`tigriver` tig is a name for tag, and also git spelled backward. River implies the visual image of flowing together of all branches. It is a three-syllable, but seems pleasant to hear.

As far as I know, it does not seem to be used by a commercial company.

Edit: Other possibilities:
- `GitRiver`
- `PanGit` (pan meaning "all-inclusive")
- `GitGo`

GitGo being my favourite.

**Comment by strk** [comment]
FWIW: I like the current name of the project.

**Comment by lcges** [comment]
I propose to write an official inquiry to trademark@sfconservancy.org
This will solve our doubts.
As for the name "Gitea" is very cool and eloquent - you can quickly remember.

@priyadarshan Git in reverse?? Poor concept.

**Comment by priyadarshan** [comment]
Sorry you did not like the first attempt. What do you think about the others?

It seems to me my [initial comment](https://github.com/go-gitea/gitea/issues/1516#issuecomment-296088035) is still pertinent.

Perhaps others would like to put more ideas on the table.

Edit: After searching, it seems the name `tig` is already been used to denote a git tool: [Tig: text-mode interface for Git](https://jonas.github.io/tig/)

**Comment by kifirkin** [comment]
@qdbdbp How about Gitter? It contains git in its name, but I think they have no problems with licensing

**Comment by jcgruenhage** [comment]
@alexmironof I don't think that having git in the name is necessarily a problem, there also is GIThub, GITlab, GITolite, GITbucket, and probably a lot more. They however don't use the git logo in theirs, which might be more of a problem.

**Comment by embik** [comment]
@alexmironof Gitter [already exists](https://gitter.im/) (and is part of GitLab).

**Comment by whilei** [comment]
I like both proposed logos so far. And those fresh color schemes! Here's another one to throw into the mix. Note the addition of the comma, too. :bowtie: 

![img_2132](https://cloud.githubusercontent.com/assets/10228550/25304744/133a7c48-2733-11e7-9857-e406bed55d7c.PNG)
![img_2133](https://cloud.githubusercontent.com/assets/10228550/25304745/134be6a4-2733-11e7-943c-e27f60bd1e3f.PNG)

**Comment by andreynering** [comment]
@whilei I liked it. Very clean.

**Comment by ulm0** [comment]
@whilei wow, i love it. Awesome work.

**Comment by whilei** [comment]
Thanks for the positive feedback. Here's another look at it. 

![img_2146](https://cloud.githubusercontent.com/assets/10228550/25313710/c0f7bf28-27f9-11e7-9700-5514a8a3c8bc.PNG)
![img_2144](https://cloud.githubusercontent.com/assets/10228550/25313693/34fe6378-27f9-11e7-8d52-2370fd9079ab.PNG)
----
**Update**: theoretical  discussion (@jcgruenhage):
![img_2150](https://cloud.githubusercontent.com/assets/10228550/25316219/4348bd78-2828-11e7-8b12-4bd9b12f0b00.PNG)
https://en.m.wikipedia.org/wiki/G
and alternatively something like below would more g-ish...? 
![img_2151](https://cloud.githubusercontent.com/assets/10228550/25316244/f999100a-2828-11e7-91bb-aac85d5290bd.PNG)

**Comment by jcgruenhage** [comment]
I can't see what the last logo should be, but I kinda like it.

**Comment by lcges** [comment]
@whilei Very cool and nice concept.
I like the first one - it is better to name it.

**Comment by whilei** [comment]
![img_2152](https://cloud.githubusercontent.com/assets/10228550/25335312/201ad1ca-28b8-11e7-9037-0cfdd9f44971.PNG)

**Comment by whilei** [comment]
![img_2153](https://cloud.githubusercontent.com/assets/10228550/25335324/2c6f09a0-28b8-11e7-9233-94f5669e6b65.PNG)

**Comment by lofidevops** [comment]
If the name does present a problem (which I agree is unlikely), or the project does not want to be tied only to Git, there is a **lot** of tea-related terminology to use as a name: *camellia*, *kukicha*, *darjeeling*, *gaiwan*, see https://en.wikipedia.org/wiki/Tea for more.

**Comment by sapk** [comment]
I think we could keep gitea for name but 
> camellia, kukicha, darjeeling, gaiwan,

could be good names for releases

**Comment by i-hate-gitea** [comment]
https://www.google.ca/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwj2-MTlzcDTAhXIqI8KHWcDDE8QjRwIBw&url=http%3A%2F%2Fwww.shutterstock.com%2Fpic-89284180%2Fstock-photo-naked-man-in-anatomical-position-isolated.html&psig=AFQjCNHhehTKqdYa1LJdWnEyhqftMfiqqw&ust=14932440643828841. Please speak English, this is the language all of us can speak and write.
2. Please ask questions or configuration/deploy problems on our Gitter channel: https://gitter.im/go-gitea/gitea
3. Please take a moment to search check that your issue doesn't already exist.
4. Please give all relevant information below for bug reports, because incomplete details will be handled as an invalid report.

**You MUST delete the content above including this line before posting, otherwise your issue will be invalid.**

- Gitea version (or commit ref):
- Git version:
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

...


## Screenshots

**If this issue involves the Web Interface, please include a screenshot**

**Comment by i-hate-gitea** [comment]
https://www.google.ca/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwj2-MTlzcDTAhXIqI8KHWcDDE8QjRwIBw&url=http%3A%2F%2Fwww.shutterstock.com%2Fpic-89284180%2Fstock-photo-naked-man-in-anatomical-position-isolated.html&psig=AFQjCNHhehTKqdYa1LJdWnEyhqftMfiqqw&ust=14932440643828841. Please speak English, this is the language all of us can speak and write.
2. Please ask questions or configuration/deploy problems on our Gitter channel: https://gitter.im/go-gitea/gitea
3. Please take a moment to search check that your issue doesn't already exist.
4. Please give all relevant information below for bug reports, because incomplete details will be handled as an invalid report.

**You MUST delete the content above including this line before posting, otherwise your issue will be invalid.**

- Gitea version (or commit ref):
- Git version:
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

...


## Screenshots

**If this issue involves the Web Interface, please include a screenshot**

**Comment by i-hate-gitea** [comment]
https://www.google.ca/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwj2-MTlzcDTAhXIqI8KHWcDDE8QjRwIBw&url=http%3A%2F%2Fwww.shutterstock.com%2Fpic-89284180%2Fstock-photo-naked-man-in-anatomical-position-isolated.html&psig=AFQjCNHhehTKqdYa1LJdWnEyhqftMfiqqw&ust=14932440643828841. Please speak English, this is the language all of us can speak and write.
2. Please ask questions or configuration/deploy problems on our Gitter channel: https://gitter.im/go-gitea/gitea
3. Please take a moment to search check that your issue doesn't already exist.
4. Please give all relevant information below for bug reports, because incomplete details will be handled as an invalid report.

**You MUST delete the content above including this line before posting, otherwise your issue will be invalid.**

- Gitea version (or commit ref):
- Git version:
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

...


## Screenshots

**If this issue involves the Web Interface, please include a screenshot**

**Comment by i-hate-gitea** [comment]
https://www.google.ca/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0ahUKEwj2-MTlzcDTAhXIqI8KHWcDDE8QjRwIBw&url=http%3A%2F%2Fwww.shutterstock.com%2Fpic-89284180%2Fstock-photo-naked-man-in-anatomical-position-isolated.html&psig=AFQjCNHhehTKqdYa1LJdWnEyhqftMfiqqw&ust=14932440643828841. Please speak English, this is the language all of us can speak and write.
2. Please ask questions or configuration/deploy problems on our Gitter channel: https://gitter.im/go-gitea/gitea
3. Please take a moment to search check that your issue doesn't already exist.
4. Please give all relevant information below for bug reports, because incomplete details will be handled as an invalid report.

**You MUST delete the content above including this line before posting, otherwise your issue will be invalid.**

- Gitea version (or commit ref):
- Git version:
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

...


## Screenshots

**If this issue involves the Web Interface, please include a screenshot**

**Comment by geek1O11** [comment]
@lunny @bkcsoft @andreynering @tboerger @agaida @jhasse @geek1011 @strk @sapk gitea is not veyr good compare to bitbucket made by unknwon

**Comment by lcges** [comment]
There any summary of the vote?

**Comment by bkcsoft** [comment]
@qdbdbp (as of this writing) your logo won. Only problem is that is integrates the Git-logo which is in violation of the Git Trademark. Mind making one that does not use said logo?  
![](https://cloud.githubusercontent.com/assets/6573628/25228803/728aef88-25d6-11e7-83d1-c003f574d6a4.PNG) was sugggested earlier as a close enough substitute 🙂

**Comment by lunny** [comment]
So let's move this to v1.3

**Comment by kolaente** [comment]
As for [3](https://github.com/go-gitea/gitea/issues/1516#issuecomment-295479783)
>Just a quick heads-up on both though... Using the Git-logo in our logo is actually not legal: https://git-scm.com/trademark ... And technically neither is using the term "Git" in the name :roll_eyes:
>
>So before starting to use this logo, we should send an email to trademark@sfconservancy.org asking if it's okey...

Any news here?

**Comment by mxmehl** [comment]
For such legal questions, the [FSFE](fsfe.org) is offering some help for free software projects like this. The lawyers answering your request will also be able to give an estimation on whether you're safe with the trademark issue.

Write to: license-questions@lists.fsfe.org

More info: https://fsfe.org/activities/ftf/licence-questions.en.html

**Comment by lafriks** [comment]
@qdbdbp could you give SVG files for logo?

**Comment by lafriks** [comment]
As @bkcsoft raised concern about git logo copyright I would propose this with same concept and colors as logo that won just without using git logo
![gitea_logo_samples](https://user-images.githubusercontent.com/165205/28467969-3afa47c4-6e3a-11e7-8a21-0ba8cc38f01a.png)

**Comment by jhasse** [comment]
Why did you do this instead of writing to trademark@sfconservancy.org?

**Comment by lafriks** [comment]
@jhasse I have created svg files for proposed one and this one, to me both ar fine :) So I will just let @go-gitea/owners to decide ;)

**Comment by jhasse** [comment]
I see :)

Btw: The 32px version in the new one is nice!

**Comment by jcgruenhage** [comment]
I like it more than the old one, mostly because the angle looks more natural, and the corners are nicer.

**Comment by lafriks** [comment]
As requested by @lunny here are some other options for small icon:

1) ![small_logo_v1](https://user-images.githubusercontent.com/165205/28494413-6178bd8e-6f36-11e7-883d-e19757d4620d.png) - :+1: 

2) ![small_logo_v2](https://user-images.githubusercontent.com/165205/28494415-6fd06ddc-6f36-11e7-85a9-a7fba7f98aee.png) - :smile: 

3) ![small_logo_v3](https://user-images.githubusercontent.com/165205/28494431-2388bf8c-6f37-11e7-9d6b-38f7bd9ec0af.png) - :tada: 

4) ![small_logo_v4](https://user-images.githubusercontent.com/165205/28494432-24e85fcc-6f37-11e7-8f36-04ccc3815178.png) - :heart:

Vote by respective reactions (vote with :-1: for same as large icon)

**Comment by twang2218** [comment]
I love the @whilei 's second design: https://github.com/go-gitea/gitea/issues/1516#issuecomment-296441534, it's very smooth, looks special, and it also looks good in small icon. Why not include that into the vote?

**Comment by lunny** [comment]
@twang2218 because @whilei 's designs are posted many days after the previous two. Except we begin a new vote, that's unfair.

**Comment by twang2218** [comment]
fine, but it looks real nice.

**Comment by MTecknology** [comment]
I'm a huge fan of the git tea that's sprouting from the tea seed. Not so much the letter g, but that next one is pretty slick. :)

**Comment by ulm0** [comment]
Ya, @whilei designs are really badass!

**Comment by whilei** [comment]
I don't mean to butt-in or cause confusion, but thought I would add a separate poll in case it may be helpful. My designs were posted 4 and 5 days (respectively) after the initial logo proposals. 

| logo | name |
| --- | --- |
| ![gitea-g](https://user-images.githubusercontent.com/10228550/28502083-49d53fe8-6fb0-11e7-90a6-82f7c8bee60f.png) | g |
| ![gitea-cup](https://user-images.githubusercontent.com/10228550/28502081-49d37e88-6fb0-11e7-92c4-559fff5907bf.png) | cup |
| ![sprout](https://user-images.githubusercontent.com/10228550/28502082-49d50258-6fb0-11e7-9e0f-f1cc844673c5.png) | sprout |

[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/01BNRDC8MJ3JWAWRV5YS07ZRK0/g)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/01BNRDC8MJ3JWAWRV5YS07ZRK0/g/vote)
[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/01BNRDC8MJ3JWAWRV5YS07ZRK0/cup)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/01BNRDC8MJ3JWAWRV5YS07ZRK0/cup/vote)
[![](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/01BNRDC8MJ3JWAWRV5YS07ZRK0/sprout)](https://m131jyck4m.execute-api.us-west-2.amazonaws.com/prod/poll/01BNRDC8MJ3JWAWRV5YS07ZRK0/sprout/vote)

**Comment by lafriks** [comment]
@whilei but still your designs got less votes than @qdbdbp so it was decided to use it and just modify not to have copyright issues with git logo.

**Comment by lcges** [comment]
[logo_gitea.svg.zip](https://github.com/go-gitea/gitea/files/1169135/logo_gitea.svg.zip)

**Comment by lcges** [comment]
In my opinion, I lost the original idea of the logo that I designed.
Now the proposal @whilei seems to me more interesting, more modern. 
Maybe as we work on the original colors of "Gitea" we will work out some good logotyp.
![gitea](https://user-images.githubusercontent.com/19255071/28571554-b493476e-7143-11e7-8618-3052956b08ad.PNG)

**Comment by lafriks** [comment]
@qdbdbp @whilei design is nice but somehow I don't like shades in background

**Comment by jonasfranz** [comment]
Which is the new logo? When we will have a decision?  I think posting more logos will not make it easier to determine which logo is "the best".

**Comment by lunny** [comment]
see #2194

**Comment by kifirkin** [comment]
@lunny I think @whilei logos looks more modern and more professionally designed than @qdbdbp and why community doesnt choose them?

The answer is simple - 2 days delay. People opened this issue and choosed from two logos first, voted and then never opened this issue again.

I think good idea to start voting from scratch and post all logos at once so everybody have some more options to choose from

**Comment by jhasse** [comment]
Before this issue, there were others with lots of ideas (#1121, #1260, #1312). To open yet another one will just result in bike-shedding.

**Comment by kifirkin** [comment]
3 issues with 6 logos, nice :rofl: Why not to gather them all in one place? And this is proof that in #1516 people just vote for 2 well known logos.

Also @qdbdbp mentioned @whilei logo is more modern and interesting.
P.S. Look carefully at `Gogs` logo - it is also well designed.
I dont want to hurt somebody feelings, but current logo seem like "Hello from 90's"

**Comment by kifirkin** [comment]
Also if you subtract downvotes from upvotes - the winner is @kolaente

**Comment by jhasse** [comment]
At some point you'll have to settle on a logo.

> And this is proof that in #1516 people just vote for 2 well known logos.

Yes, because those were the two most popular suggestions at the time.

> Also if you subtract downvotes from upvotes - the winner is @kolaente

Voting is over, the current state isn't representative anymore (for example people who dislike the logo on try.gitea.io might come here and downvote it).

**Comment by kifirkin** [comment]
Also only clear thinking mades my decision to move from gogs to gitea, because it seems unmaintained, but community choose gogs(it has about 20k stars)

**Comment by jhasse** [comment]
> The state of downvotes was correct at the time when pull request arrived

I didn't know that. I guess if down-votes were to be substracted from the voting, that should have been made clear in the beginning. Without explicit mentioning I wouldn't expect it.

> If you think rationally not only community votes makes good choice

Yes, so?

> but community choose gogs(it has about 20k stars)

Gogs was around a lot longer that's why it has more stars.

**Comment by lcges** [comment]
GOGS is a older project - compared to him, the new project is stupidity.
Do you think people will not go to Gitea because there is a "different" logo than Gogs?

**Comment by kifirkin** [comment]
> Gogs was around a lot longer that's why it has more stars.

It has more start because of nice logo :wink:

**Comment by jhasse** [comment]
> It has more start because of nice logo :wink:

No, it didn't even have the current logo from the beginning.

**Comment by kifirkin** [comment]
Yes, previous logo seems more popular :smile: 
http://www.timqian.com/star-history/#go-gitea/gitea

**Comment by jhasse** [comment]
Why do you think that? I don't see any decline of the gradient since July.

**Comment by kifirkin** [comment]
From start up it has a huge gain.

So I think we should stop flooding about personal decisions. My decision is good looking, professianally designed and modern logo

**Comment by lesderid** [comment]
@alexmironof The decision was made to use the logo with the most votes. If you think it's so bad that there needs to be a new vote, open a new issue with objective arguments against the current logo. If enough people agree with you, a new vote could be proposed, but I don't see it happening.  
People chose for the current logo, and if anything is bad for brand identity, it's changing logos every few months.

**Comment by kifirkin** [comment]
@lesderid If anythings is bad for brand identity it is only a bad brand identity position.
Like choosing not the best logo available or bad contribution model

**Comment by ptman** [comment]
Isn't it very simple to override the logo in custom/ ? Please focus your effort on getting a new release out instead of logo bikeshedding.

**Comment by lcges** [comment]
Please suggest some modern logo that you think is correct.
At this time it's just trolling.
We focus on important things - the logo theme was already considered.

If you have a problem with this, maybe it's better to be with Gogs because it has a better logo.

I think Gitea is already bright years ahead of Gogs with an "ugly" logo ...

**Comment by kifirkin** [comment]
Ive suggested it before

**Comment by tboerger** [comment]
The vote is over and a logo have been taken and integrated.
